packages/components/src/Toast/Toast.mdx

```diff
@@ -0,0 +1,97 @@
+---
+name: Toast
+menu: Components
+route: /components/toast
+---
+
+import { Playground, Props } from "docz";
+import { ComponentStatus } from "@jobber/docx";
+import { useRef } from "react";
+import { Button } from "@jobber/components/Button";
+import { showToast } from ".";
+import { Toast } from "./Toast";
+
+# Toast
+
+<ComponentStatus stage="ready" responsive="no" accessible="no" />
+
+Toasts are short, temporary messages used to inform users that a process was
+performed. They provide visual feedback on the outcome of an action, and require
+minimal user interacction.
+
+Toast, unlike common React components is not a component that will be added to
+your view. Instead, toast is used by importing and calling the `showToast()`
+function.
+
+```ts
+import { showToast } from "@jobber/components/Toast";
+```
+
+<Playground>
+  <Button
+    label="show toast"
+    onClick={() =>
+      showToast({
+        message: "Toast has been toasted"
+      })
+    }
+  />
+</Playground>
+
+## Design & Usage Guidelines
+
+When contributing to, or consuming the Toast component, consider the following:
+
+- Keep text labels as short as possible. They should be clear and concise, and
+  should not take up more than one line.
+- Use the pattern: noun + verb.
+  - Examples: `Job created`, `Quote approved`, `Client archived`.
+- No need to use "Successfully", should be implicit in the toast.
+- Don't use `Dismiss` or `Cancel` as an action label.
+  - Examples of action labels: `Undo`, `View`, `Refresh`.
+
+## Props
+
+<Props of={Toast} />
+
+## Variations
+
+Toasts have 3 variations, `default`, `success`, and `error`. Each should be used
```

Comment:
How do you feel about `info` instead of `default`?

Reply:
lets change `default` to `info` and default to `info`